pkg/logging/http.go

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) The Thanos Authors.
+// Licensed under the Apache License 2.0.
+
+package logging
+
+import (
+	"fmt"
+
+	"net/http"
+	"time"
+
+	"github.com/go-kit/kit/log"
+	package_http "github.com/thanos-io/thanos/pkg/server/http"
+)
+
+type ServerMiddleware struct {
```

Comment:
```suggestion
type HTTPServerMiddleware struct {
```

?

Reply:
lol, @kakkoyun suggested to change it to ServerMiddleware :stuck_out_tongue: 
Anyways, I would change it!

Reply:
Done